src/main/java/seedu/address/logic/commands/AddGroupCommand.java

```diff
@@ -85,11 +89,26 @@ public CommandResult execute(Model model) throws CommandException {
 
             Student editedStudent = createEditedStudent(studentToEdit, allocDescriptor);
             toAdd.addStudent(editedStudent.getId());
+            addedStudents.add(editedStudent);
             model.setStudent(studentToEdit, editedStudent);
         }
 
         model.addGroup(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+        return new CommandResult(formatSuccessMessage(addedStudents));
+    }
+
+    public String formatSuccessMessage(List<Student> addedStudents) {
+        String groupAddedMessage = String.format(MESSAGE_SUCCESS, toAdd.name);
+
+        if (addedStudents.isEmpty()) {
+            return groupAddedMessage;
+        }
+
+        List<String> studentNames = addedStudents.stream()
+                .map(student -> student.getName().fullName)
+                .collect(Collectors.toList());
```

Comment:
Can use `Collectors.joining(", ")` here instead of doing the join in the next line.